server/actions/elections.go

```diff
@@ -56,7 +70,13 @@ func EditElection(c *gin.Context) {
 	defer database.ReleaseDB()
 	if err := db.First(&election).Error; err != nil {
 		fmt.Println(err)
-		c.String(http.StatusBadRequest, "invalid election specified")
+		c.String(http.StatusBadRequest, "400 Bad Request: Invalid election specified")
```

Comment:
Should they message contain the status code?

Reply:
I've seen examples of both having the status in the code or not. The purpose for it was mostly to make debugging somewhat nicer, since it was clearer what went wrong. But tbf, as long as the error messages are good, this shouldn't be a problem.